### PR TITLE
GRW-1313/chore/update dependencies

### PR DESCRIPTION
**Pull request by alebedev**
## GRW-1313/chore/update dependencies

Update racoon dependencies across the board

## Justify why they are needed

Maintenance day activity

Verified following major version updates
- jest
- jsdom

None of above break our code or require config changes

## Checklist before requesting a review

- [x] I have performed a self-review of my code

[GRW-1313]: https://hedvig.atlassian.net/browse/GRW-1313?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1 @@
+save-prefix ""
```
Comment (robinandeer) [inline .yarnrc:1]: Maybe we can add a comment here to explain why? Like:

```
# Docs: https://classic.yarnpkg.com/en/docs/yarnrc

# Save exact dependency version by default
save-prefix ""
```

File: apps/onboarding/package.json [hunk 2]
```diff
@@ -32,14 +32,14 @@
     "framer-motion": "^6.3.0",
     "graphql": "16.3.0",
     "graphql-tag": "^2.12.6",
-    "jsdom": "^19.0.0",
+    "jsdom": "^20.0.0",
```
Comment (robinandeer) [inline apps/onboarding/package.json:35]: When upgrading major version, please made a note in the PR that there's no relevant breaking changes :)

File: apps/onboarding/package.json [hunk 3]
```diff
@@ -49,29 +49,29 @@
...
     "@types/node": "^17.0.24",
-    "@types/react": "^17.0.39",
+    "@types/react": "17.0.48",
     "cypress": "^9.5.4",
-    "eslint": "^8.8.0",
-    "eslint-plugin-cypress": "^2.12.1",
-    "jest": "^27.5.1",
+    "eslint": "8.21.0",
+    "eslint-plugin-cypress": "2.12.1",
+    "jest": "28.1.3",
```
Comment (robinandeer) [inline apps/onboarding/package.json:66]: Also major version, we should make a note in the PR description :)

File: packages/scripts/package.json [hunk 4]
```diff
@@ -13,21 +13,21 @@
...
-    "eslint-config-next": "12.2.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-import-resolver-typescript": "^2.5.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-testing-library": "^5.0.5",
+    "@typescript-eslint/eslint-plugin": "5.32.0",
+    "@typescript-eslint/parser": "5.32.0",
+    "eslint-config-next": "12.2.3",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-import-resolver-typescript": "3.4.0",
```
Comment (robinandeer) [inline packages/scripts/package.json:20]: Major version :)

**Review by robinandeer — COMMENTED** [review]
Great job here! 🙌

Let's just add some comments in the PR descriptions about the dependencies where we upgrade major versions -- just that we looked through the changelog and didn't find and relevant breaking changes. That makes sense right? :)

[bow](https://app.graphite.dev/macros/HedvigInsurance)

File: packages/scripts/package.json (hunk 4 above)
Comment (alebedev) [reply packages/scripts/package.json:20]: Removed it altogether - we get correct version from transitive reps

**Review by robinandeer — APPROVED** [review]

**Review by robinandeer — COMMENTED** [review]
Wow so many changes -- let's make sure to review this when we merge :D

**Pull request merged**